dist/scripts/api-console.js

```diff
@@ -517,8 +517,12 @@
             jQuery($this).addClass('raml-console-is-active');
             $scope.showPanel = true;
 
-            $location.hash(hash);
-            $anchorScroll();
+            setTimeout(function () {
```

Comment:
Can't we use `$timeout` instead? or do we need to run this outside of an angular digest loop? If that's the case let's document in the code the reason why are we using `setTimeout`